cloudbuild.yaml

```diff
@@ -43,62 +43,64 @@
-#  waitFor: ['-']
-
-#- id: 'Docker Image: openmatch-go-func-simple'
-#  name: gcr.io/cloud-builders/docker
-#  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-go-func-simple:${_OM_VERSION}-${SHORT_SHA}', 'examples/functions/golang/manual-simple']
-#  waitFor: ['-']
-
-#- id: 'Docker Image: backendclient'
-#  name: gcr.io/cloud-builders/docker
-#  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-backendclient:${_OM_VERSION}-${SHORT_SHA}', 'examples/backendclient']
-#  waitFor: ['-']
-
-#- id: 'Docker Image: clientloadgen'
-#  name: gcr.io/cloud-builders/docker
-#  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-clientloadgen:${_OM_VERSION}-${SHORT_SHA}', 'test/cmd/clientloadgen']
-#  waitFor: ['-']
+- id: 'Docker Image: open-match-base-build'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'open-match-base-build', '-f', 'Dockerfile.base-build', '.']
+  waitFor: ['-']
 
-#- id: 'Docker Image: frontendclient'
-#  name: gcr.io/cloud-builders/docker
-#  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-frontendclient:${_OM_VERSION}-${SHORT_SHA}', 'test/cmd/frontendclient']
-#  waitFor: ['-']
+- id: 'Docker Image: backendapi'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-backendapi:${_OM_VERSION}-${SHORT_SHA}', 'cmd/backendapi']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: frontendapi'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-frontendapi:${_OM_VERSION}-${SHORT_SHA}', 'cmd/frontendapi']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: mmforc'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-mmforc:${_OM_VERSION}-${SHORT_SHA}', 'cmd/mmforc']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: mmlogicapi'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-mmlogicapi:${_OM_VERSION}-${SHORT_SHA}', 'cmd/mmlogicapi']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: Evaluator Simple'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-evaluator-simple:${_OM_VERSION}-${SHORT_SHA}', 'examples/evaluators/golang/simple']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: openmatch-cs-func-simple'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-cs-func-simple:${_OM_VERSION}-${SHORT_SHA}', 'examples/functions/golang/manual-simple']
+  waitFor: ['Docker Image: open-match-base-build']
+
+- id: 'Docker Image: openmatch-go-func-simple'
+  name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openmatch-go-func-simple:${_OM_VERSION}-${SHORT_SHA}', 'examples/functions/golang/manual-simple']
+  waitFor: ['Docker Image: open-match-base-build']
+
```

Comment:
Does building mechanism for php / python function containers stay the same? Dont see them included in the steps here.

Reply:
https://github.com/GoogleCloudPlatform/open-match/pull/129/files